### PR TITLE
## JAVASCRIPT: Weather App

### PR DESCRIPTION
**Pull request by FrederickMih**
## THE WEATHER APP
 In this branch,
- I installed and configured webpack
- I wrote functions that fetch some vital information from a weather map API such as the temperature(degree celsius or Fahrenheit ), humidity, weather description from a particular city of a country.
- I equally make use of the giphy API images to describe how the weather condition may look.

### CODE REVIEW

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ⚠️ 🚧
@FrederickMih, Weldone for the job done. Almost all the requirement is met but few changes is requested to get your project approved.

- [ ] It changes its appearance based on the received data ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))
- [ ] kindly add the screenshot of your app to your readme and in your setup section kindly use 'npm run start' to run your app.
- [ ] I can't run your app with the script provided in your `package.json` ,Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)).please replace the script wth this :
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

### Once specific changes made kindly request for another review

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS : CHANGES REQUIRED ♻️ 
@FrederickMih  Good day, thanks for implementing some part of the requested cahnges,however changes is required to get your project approved
### highlight

- [x] kindly add the screenshot of your app to your readme 👍🏻 


### To improve
- [ ] It changes its appearance based on the received data ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app)).

- [ ] Kindly add `webpack.config.js` to your root file and remove `webpack.common.js`,`webpack.dev.js`,`webpackprod.js`. here is how to install webpack.config.js [link](https://webpack.js.org/guides/getting-started/#using-a-configuration) and you can replace ` "start": "webpack-dev-server --mode development --open" with
`   `"start": "webpack serve --mode development --open"` as its noticed its not working
- [ ] Your getting started section still missing  `npm start` kindly fix it

.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @FrederickMih  👋

**You've done pretty well here fixing a few issues raised in the previous review. 👌**

However, let's make a couple of changes.

- [ ] After the weather data loads the form to input a new city gets hidden due to a styling bug. Please check and fix it.

- Please refer to the other comment on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: README.md [hunk 1]
```diff
@@ -1 +1,61 @@
...
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- `git clone git@github.com:FrederickMih/WEATHER-APP.git`
+- `cd weather-app`
+- `npm install`
+- open `dist/index.html` 
+
```
Comment (mikenath223) [inline README.md]: - [ ] As requested in the previous review please add `npm start` after `npm install` instruction as the command ensures webpack compiles the project before opening it.

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**